watertap/property_models/tests/test_coag_prop_pack.py

```diff
@@ -222,3 +222,53 @@ def test_solution(self, coag_obj):
         assert value(model.fs.stream[0].conc_mass_phase_comp['Liq', 'TSS']) == pytest.approx(9.932509,  rel=1e-4)
         assert value(model.fs.stream[0].conc_mass_phase_comp['Liq', 'TDS']) == pytest.approx(9.932509,  rel=1e-4)
         assert value(model.fs.stream[0].conc_mass_phase_comp['Sol', 'Sludge']) == pytest.approx(2600,  rel=1e-4)
+
+
+class TestCoagulationPropPackFailures():
+    @pytest.fixture(scope="class")
+    def coag_obj_fail(self):
+        model = ConcreteModel()
+        model.fs = FlowsheetBlock(default={"dynamic": False})
+        model.fs.properties = CoagulationParameterBlock()
+
+        return model
+
+    @pytest.mark.unit
+    def test_default_scaling(self, coag_obj_fail):
+        model = coag_obj_fail
+
+        model.fs.stream = model.fs.properties.build_state_block([0], default={})
+
+        # call scaling without setting defaults
+        iscale.calculate_scaling_factors(model.fs)
+
+        # check that all variables have scaling factors
+        unscaled_var_list = list(iscale.unscaled_variables_generator(model))
+        assert len(unscaled_var_list) == 0
+
+        # check that all constraints have been scaled
+        unscaled_constraint_list = list(iscale.unscaled_constraints_generator(model))
+        assert len(unscaled_constraint_list) == 0
+
+        # check if any variables are badly scaled
+        badly_scaled_var_list = list(iscale.badly_scaled_var_generator(model, large=1e2, small=1e-2))
+        if len(badly_scaled_var_list) != 0:
+            lst = []
+            for (var, val) in badly_scaled_var_list:
+                lst.append((var.name, val))
+                print(var.name, var.value)
+            print("The following variable(s) are poorly scaled: {lst}".format(lst=lst))
+        assert len(badly_scaled_var_list) == 0
```

Comment:
By default pytest will automatically display values used for `assert`, so this could be simplified as e.g.:

```py
badly_scaled_var_values = {
    var.name: val
    for (var, val) in iscale.badly_scaled_var_generator(
        model, large=1e2, small=1e-2
    )
assert not badly_scaled_var_values
```